HeavyFlavorAnalysis/RecoDecay/BuildFile.xml

```diff
@@ -0,0 +1,27 @@
+<use   name="DataFormats/ParticleFlowCandidate"/>
+<use   name="DataFormats/PatCandidates"/>
+<use   name="DataFormats/RecoCandidate"/>
+<use   name="DataFormats/TrackReco"/>
+<use   name="DataFormats/VertexReco"/>
+<use   name="FWCore/Framework"/>
+<use   name="FWCore/MessageLogger"/>
+<use   name="PhysicsTools/CandUtils"/>
+<use   name="RecoVertex/KalmanVertexFit"/>
+<use   name="RecoVertex/KinematicFit"/>
+<use   name="RecoVertex/KinematicFitPrimitives"/>
+<use   name="RecoVertex/VertexPrimitives"/>
+<use   name="TrackingTools/PatternTools"/>
+<use   name="TrackingTools/Records"/>
+<use   name="TrackingTools/TransientTrack"/>
+<flags CXXFLAGS="-Wno-delete-non-virtual-dtor"/>
```

Comment:
I do not remember, but apparently the problem disappeared.
Probably it has been fixed since my first test with older CMSSW

On Tue, 23 Aug 2016, Slava Krutelyov wrote:

> 
> In HeavyFlavorAnalysis/RecoDecay/BuildFile.xml:
> 
> > +<use   name="DataFormats/ParticleFlowCandidate"/>
> > +<use   name="DataFormats/PatCandidates"/>
> > +<use   name="DataFormats/RecoCandidate"/>
> > +<use   name="DataFormats/TrackReco"/>
> > +<use   name="DataFormats/VertexReco"/>
> > +<use   name="FWCore/Framework"/>
> > +<use   name="FWCore/MessageLogger"/>
> > +<use   name="PhysicsTools/CandUtils"/>
> > +<use   name="RecoVertex/KalmanVertexFit"/>
> > +<use   name="RecoVertex/KinematicFit"/>
> > +<use   name="RecoVertex/KinematicFitPrimitives"/>
> > +<use   name="RecoVertex/VertexPrimitives"/>
> > +<use   name="TrackingTools/PatternTools"/>
> > +<use   name="TrackingTools/Records"/>
> > +<use   name="TrackingTools/TransientTrack"/>
> > +<flags CXXFLAGS="-Wno-delete-non-virtual-dtor"/>
> 
> which other packages?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the
> thread.[AE4A-Qd1bvpC5fFHHr40QwH6UAcsDHhmks5qiyC8gaJpZM4JlnI8.gif]
> 
> 
>